rmc-docs/src/cargo-rmc.md

```diff
@@ -1,52 +1,41 @@
 # RMC on a package
 
-> RMC currently ships with a `cargo-rmc` script, but this support is deeply limited (e.g. to a single crate).
-> This will be corrected soon, and this documentation updated.
-> In the meantime, we document the current build process for a larger project with dependencies here.
-
-To build a larger project (one with dependencies or multiple crates) with RMC, you currently need to:
-
-1. Build the project with an appropriate set of flags to output CBMC "symbol table" `.json` files.
-2. Link these together into a single "goto binary", with appropriate preprocessing flags.
-3. Directly call CBMC on this resulting binary.
-
-We give an example of this kind of script, with explanations, below.
+RMC currently ships with a `cargo-rmc` script, but this support is limited. If you find any issue, please [filed a bug report](https://github.com/model-checking/rmc/issues/new?assignees=&labels=bug&template=bug_report.md).
 
 # Building and running
 
-Let's assume you have a project you can build with `cargo build` and you've written a proof harness somewhere in it that you want to run RMC on:
+Let's assume you have a library project that you can build with `cargo build` and you've written a proof harness somewhere in it that you want to run RMC on:
 
 ```rust
-#[no_mangle]
 #[cfg(rmc)]
+#[rmc::proof]
```

Comment:
I think we should add a section before this one that introduces the `#[rmc::proof]` annotation.

Reply:
What if I just remove this example. It's not adding much value anymore.

Reply:
And only leave the `cargo rmc --function my_harness` command in this section? If so, my understanding is that the command wouldn't work without annotating the function with either `#[no_mangle]` or `#[rmc::proof]`.